### DISCUSSION TITLE
java agent plugin for brpc3

### DISCUSSION THREAD

**Discussion by wangsan** [apache/skywalking#9564] [General]
hi all 
sw 9.2+sw-agent 8.12 now support brpc-java V2.5；
but not support bprc3.x  as brpc 3 client has been refactored

**Comment by wu-sheng** [comment]
Every plugin is contributed by the community, we have provided very detailed plugin dev doc, I think it is not hard to build one with tests. If you want, you could contribute it to the official upstream.
